community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -139,6 +139,10 @@
     public static final Setting<Long> transaction_start_timeout =
             setting( "transaction_start_timeout", DURATION, "1s" );
 
+    @Description("Location of the database plugin directory. Compiled Java JAR files that contain database " +
+                 "procedures will be loaded if they are placed in this directory.")
+    public static final Setting<File> plugin_dir = setting("dbms.plugin.directory", PATH, "plugins" );
```

Comment:
why do we still call folder for procedures `plugins`? 
This name gives impression that its just another place for current unmanaged extensions and i had an impression we want to move away from that.